data/transport missions.txt

```diff
@@ -1951,3 +1951,244 @@ ship "Science Drone"
+		fleet "Large Southern Pirates"
+	npc
+		government "Pirate"
+		personality staying harvests plunders
+		system "Pherkad"
+		fleet "Small Southern Pirates" 3
+	npc
+		government "Free Worlds"
+		personality staying heroic
+		system "Kappa Centauri"
+		fleet "Large Free Worlds"
+	npc
+		government "Free Worlds"
+		personality staying heroic
+		system "Yed Prior"
+		fleet "Large Free Worlds"
+	npc
+		government "Free Worlds"
+		personality staying heroic
+		system "Pherkad"
+		fleet "Large Free Worlds"
+	on stopover
+		dialog `The last of the defenses are quickly unloaded from your ship, put into place, and activated. One of the pirate ships attacking the system attempts to follow you down to the surface, but the fire coming from the newly installed spaceport turrets forces the pirate to turn away and flee for orbit. Time to return to <destination>.`
+	on complete
+		payment
+		payment 5000
+		dialog `The Free Worlds captain thanks you for helping to fortify the frontier worlds, and pays you the promised <payment>.`
+
+
+
+mission "Core Blockade"
+	minor
+	name `Bad Harvest`
+	description `The pirate world of <destination> is experiencing a famine. Warlords of the planet have threatened the Syndicate to give them food, and the Syndicate has agreed. Deliver <cargo> to the world for <payment>.`
+	source
+		near "Almach" 2 4
+	destination "Deadman's Cove"
+	cargo "food" 287
+	to offer
+		random < 20
+		"combat rating" > 400
+	
+	on offer
+		conversation
+			`In the <origin> spaceport, you are approached by a Syndicate employee. By the looks of his uniform, he works for Syndicated Security. "Hello, Captain. You're a compitent pilot, correct?" You nod your head. "Excellent. The Syndicate needs <cargo> transported to <destination>. The pirate world has seen a particularly bad harvest this past season, so much of the population is without sufficient food. Pirate warlords have threatened to begin raiding our freighters and frontier worlds if we don't provide them with food. It's cheaper to do what they want, so we've agreed to the deal, but we don't want to risk any of our own ships being destroyed. The Syndicate will pay you <payment> should you accept and complete this job."`
+			choice
+				`	"Alright, I'll get my ship ready."`
+					accept
+				`	"Sorry, but I have better things to be doing."`
+					decline
+	
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+		system destination
+		fleet "Large Core Pirates" 2
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+		system "Schedar"
+		fleet "Large Core Pirates"
+		fleet "Small Core Pirates" 3
+	
+	on complete
+		payment
+		payment 10000
+		dialog `Multiple bands of pirate each run by a different warlord are waiting in the spaceport when you land. They take the food from your cargo hold like animals, leaving quite the mess behind. When you return to your ship, you find that the Syndicate has deposited <payment> into your account.`
```

Comment:
`bands of pirate` -> `bands of pirates`

Should put commas in between `each` and `run`; and `warlord` and `are`. Otherwise, it gets confusing; sounds like the pirate bands are running by the warlords.